src/components/RenderForm.js

```diff
@@ -14,19 +14,28 @@ export default function RenderForm (props) {
   let current_schema = props.schema;
 
   if (current_schema === '') {
-    return <body> A schema has not been selected yet </body>
+    return <div> Please select a schema. </div>
   };
 
   const preProcessing = (key) => {
     /*
     PreProcessing for schema validation
       Adds id (same as $schema) field to address ajv5 validation and jsonschema 2020-12 compatibility
+      Grays out const fields (prop.readOnly) and autofills the field with the const value (prop.default)
     */
-    // const key = schema_path
     const schema = schema_map[key]
     if (schema.$schema !== undefined) {
       schema.id = schema.$schema;
     }
+
+    const properties = schema.properties
+    for (const [name, prop] of Object.entries(properties)) {
```

Comment:
You'll need to iterate through all the nested properties and through all the entries in defs. This method should be fine for now, but there might be a way to add a ui-schema to the rjsf Form class to gray out the constant fields. You can see this thread for how to do a nested loop in javascript objects: https://stackoverflow.com/questions/8085004/iterate-through-nested-javascript-objects